src/main/java/org/gradle/wrapperupgrade/UpgradeWrapper.java

```diff
@@ -201,6 +203,14 @@ private void gitCreatePr(Params params, String prTitle, String prBody) throws IO
             if (!labels.isEmpty()) {
                 pr.addLabels(labels.toArray(new String[0]));
             }
+            List<String> reviewers = upgrade.getOptions().getReviewers().get();
+            if (!reviewers.isEmpty()) {
+                List<GHUser> githubReviewers = mapToGHUsers(params, reviewers);
```

Comment:
I think you forgot to apply the reviewers to the PR with `pr.requestReviewers(githubReviewers)`, same for the assignees